### ISSUE TITLE
Deleting contents of `card.js` in the test project doesn't give canvas error

### ISSUE THREAD

**Issue by maltenuhn** [concrete-utopia/utopia#1309]
STR
1. In the test project, delete the contents of card.js (don't save)
2. Behold the canvas

Expected: error
Observed: no error

**Comment by enidemi** [comment]
I see this error when deleting the contents of `card.js`. Is it possible that it was fixed as part of a different bugfix?

<img width="1440" alt="Screenshot 2021-05-28 at 13 54 07" src="https://user-images.githubusercontent.com/4403069/119980193-3d694c80-bfbc-11eb-91ad-a45e981f13cd.png">

**Comment by maltenuhn** [comment]
Yes! Alas, now the error refers to `UtopiaSpiedFunctionComponent` and says
it's coming from Utopia itself...

**Comment by enidemi** [comment]
I actually see it frequently, but not always, that deleting the `card` content then undoing changes doesn't update the canvas and keeps this error message here.

**Comment by maltenuhn** [comment]
Yes; I wonder if it's also related to the issue @seanparsons you're tackling, but I'm going to try and create some additional error cases around 'undo' (in code editor) and commenting things out; I suspect there's still a bug or two hiding there.
